src/http.rs

```diff
@@ -53,50 +52,52 @@ async fn run(
 
     // GET /hd/:fingerprint
     let hd_wallet_handler = warp::get()
-        .and(warp::path!("hd" / Fingerprint))
```

Comment:
All these endpoints should be renamed from `/hd/` to `/wallet/`.

Once we add backwards compatibility for xpubs, this should support both `/wallet/<descriptor-checksum>` and `/wallet/<bip32-extkey-fingerprint>`

Reply:
I renamed `/hd` -> `/wallet` but am deferring the backwards compatibility to a future PR